examples/meteringlab/c2/dataloaders/n4.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: n4-dataloader
+  namespace: default
+  labels:
+    app: dataloader
+spec:
+  template:
+    metadata:
+      labels:
+        app: job
+    spec:
+      volumes:
+        - name: c1n4-connection-bundle
+          secret:
+            secretName: c1n4-connection-bundle
+      containers:
+      - name: dataloader
+        image: scylladb/scylla-bench:latest
+        volumeMounts:
+        - name: c1n4-connection-bundle
+          mountPath: /var/run/secrets/kubernetes.io/connbundle
+          readOnly: true
+        command:
+          - "/bin/bash"
+          - "-c"
+          - 'scylla-bench -tls true -workload sequential -mode write -nodes n1,n2,n3 -clustering-row-size 16 -validate-data -cloud-config-path /var/run/secrets/kubernetes.io/connbundle'
```

```suggestion
          - 'scylla-bench  -workload sequential -mode write -clustering-row-size 16 -validate-data -cloud-config-path /var/run/secrets/kubernetes.io/connbundle/sc1.ml.virgil.scylla-operator.scylladb.com'
```
Bundle secret contains a key per dns domain under which ScyllaCluster is exposed, volume is mounted as directory having keys as files.